doc/classes/@GlobalScope.xml

```diff
@@ -995,6 +995,18 @@
 				[/codeblock]
 			</description>
 		</method>
+		<method name="randb">
+			<return type="bool" />
+			<param index="0" name="chance" type="float" />
+			<description>
+				Returns a random boolean with a bias of [param chance] to be [code]true[/code] and [code]false[/code] otherwise.
+				[codeblocks]
+				[gdscript]
+				randb(0.5) # Returns true at the param chance of 0.5.
+				[/gdscript]
+				[/codeblocks]
```

Comment:
```suggestion
				Returns either [code]true[/code] or [code]false[/code] at random. [param true_chance] should be a number between [code]0.0[/code] and [code]1.0[/code] (inclusive). The greater [param true_chance] is, the more likely this function returns [code]true[/code].
				[codeblocks]
				[gdscript]
				rand_bool(0.0) # Returns false, all the time.
				rand_bool(0.5) # Returns true, 50% of the time.
				rand_bool(1.0) # Returns true, all the time.
				[/gdscript]
				[csharp]
				GD.RandBool(0.0); // Returns False, all the time.
				GD.RandBool(0.5); // Returns True, 50% of the time.
				GD.RandBool(1.0); // Returns True, all the time.
				[/csharp]
				[/codeblocks]
```
With the new range(s)

Reply:
Note that given the current implementation in this PR `randb(1.0)` can return `false`. This will happen when `randf()` returns `1.0`. It's very unlikely but possible.

For a guaranteed true one would need to call `randb(chance)` with `chance > 1.0`.

Reply:
I'd say to just change it to `randf() <= p_true_chance`, the actual difference *except* for the upper end doesn't really matter (assuming `randf` is *guaranteed* to produce strictly in [0,1])

Reply:
Doesn't seem too bad to just directly return true if the passed var is >= 1.0. While it's an extra comparison it does allow to skip the RNG routine.

Reply:
I do agree with having a check to prevent unnecessary use of RNG. It is an easy addition.

Should I refactor with the above changes and new method/variable naming?

Reply:
Please do :) I think a warning isn't needed, there isn't one for `rand_range`

Reply:
If this was solved before, it got reverted now to an extent.